src/Savegame/Soldier.cpp

```diff
@@ -87,6 +88,7 @@ Soldier::Soldier(RuleSoldier *rules, Armor *armor, int id) :
 			_look = (SoldierLook)RNG::generate(0,3);
 			_name = (_gender == GENDER_FEMALE) ? "Jane" : "John";
 			_name += " Doe";
+			_callsign = "Unknown";
```

Comment:
Should be there empty too?

Reply:
This should hopefully never happen, but yeah... I would leave it empty here too.